xrpl4j-binary-codec/src/main/java/org/xrpl/xrpl4j/codec/binary/types/STObjectType.java

```diff
@@ -98,7 +105,29 @@ public STObjectType fromJson(JsonNode node) {
         continue;
       }
 
-      JsonNode fieldNode = node.get(fieldName);
+      JsonNode fieldNode;
+      // rippled expects signers canonically based on address
+      if (fieldName.equals("Signers")) {
+        final AddressCodec addressCodec = new AddressCodec();
+        ArrayNode arrayNode = (ArrayNode) node.get(fieldName);
+        List<JsonNode> jsonNodeList = new ArrayList<>();
```

Comment:
We need unit test coverage of this in addition to the ITs...that codecov tool in IDEA is your friend! :)